projects/ccf-rui/src/app/app.component.ts

```diff
@@ -66,6 +92,29 @@ export class AppComponent implements OnDestroy, OnInit {
       },
       duration: this.consentService.consent === 'not-set' ? Infinity : 3000
     });
+
+    this.theming.setTheme(`${this.theme}-theme-light`);
+
+    // if (window.matchMedia) {
```

Comment:
You can go ahead and remove this code to make sonar happy. We can always pull it from history or the EUI.